guides/common/modules/proc_configuring-an-external-dhcp-server.adoc

```diff
@@ -22,11 +15,11 @@ If the `dhcp-no-override` setting is disabled, hosts fetch the bootloader and it
 include::snip_firewalld.adoc[]
 
 .Procedure
-. On your {EL} host, install the ISC DHCP Service and Berkeley Internet Name Domain (BIND) utilities packages:
+. On your external DHCP host, install the ISC DHCP Service and Berkeley Internet Name Domain (BIND) utilities packages:
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
-# {client-package-install-el8} dhcp-server bind-utils
+# {package-manager} install {dhcp-server-package} {bind-package}
```

Comment:
Where does "package-manager" come from? Do we only document/allow running external DHCP on the same base OS as Foreman Server?

I can create a follow-up PR to unset "package-manager" after using it in various procedures. If possible, it is defined within "attributes-*.adoc".

Reply:
> Do we only document/allow running external DHCP on the same base OS as Foreman Server?

No, but it's reasonable to expect that users use the same OS for their servers. Most users try to limit the different number of operating systems and if you install Foreman on Debian, you're most likely a Debian shop. Expecting them to install an external server on EL 8 is then unlikely.

> I think the issues comes from "we cannot use project-package-install because Satellite uses "satellite-maintain ...", and that's obviously not present on hosts that are not Satellite Server/Capsule Servers yet.

Yes, that's correct.

Reply:
I'm looking more at https://github.com/theforeman/foreman-documentation/commit/fd8ed92fd762376cf0ae11e16b5e91b8280ec787 and I'm sorry I didn't get to review it then, but I think it made things worse. IMHO we need attributes for the server package manager. I'll work on reintroducing those.

Reply:
There are only a few select instances that actually require a third way to install packages next to client-pkg-install and project-pkg-install. We should use a meaningful third attribute for this where required for Satellite. All other build targets can probably go with "third-attribute: {project-pkg-install}".

Reply:
We also have `{server-package-manager} upgrade` in my current patch - would you prefer separate attributes for install/upgrade (we can get away with hardcoding `apt update` since it'll always be Debian-only because yum/dnf has no equivalent) or `{project-pkg-manager}` / `{project-package-manager}`?

Reply:
I don't yet have a better alternative to "server-package-manager", so I think it's fine.

Reply:
From triage: let's consider the name of the attribute non-blocking for your PR.